### ISSUE TITLE
Megabits reads instead of megabytes with Time NetSpeed Monitor?

### ISSUE THREAD

**Issue by kratoz29** [visnkmr/visnkmr#4]
I liked more how the reads were shown in the previous version as megabit, now they changed to megabytes with the version 1.2G and I think megabits works better for NetSpeed metrics, is it possible to going back to it, or maybe making this a toggle? 

Also I think it is not showing my free RAM withing the  "Useful Information Screen" my device is a Nvidia Shield TV Pro 2019 running Shield Experience 8.2.3.

**Comment by multichoyce** [comment]
Same, Do give as an option to switch between the two units, I had to roll back just for this

**Comment by kratoz29** [comment]
> Same, Do give as an option to switch between the two units, I had to roll back just for this

Yeah, that and the lack of showing the system RAM made me roll back as well, the only downside is that I need to keep in mind to not hit update each time I update my apps.
